index.html

```diff
@@ -4,6 +4,8 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+	<h1>Sam Huang</h1>
```

Comment:
Looks like you're using the default tab setting of 4 spaces. Please change to two!